### PR TITLE
[Review required] Module and filename renaming

### PR DESCRIPTION
**Pull request by RaananW**
A new unified naming convension for our filenames, including fallbacks to support the old files:

[modulename].js and [modulename].min.js , in [modulename] directory.

After merging there are many files to delete. If merged, it is recommended to 
1. delete all .js and all .d.ts files in /dist/preview release/  
2. Rebuild all modules
3. Commit the untracked files, remove unneeded files from the repository.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -180,8 +180,8 @@ dist/preview release/package/
 /Viewer/dist/viewer.js
 /Viewer/dist/viewer.min.js
 dist/preview release/viewer/babylon.d.ts
-dist/preview release/viewer/babylonjs.loaders.d.ts
-dist/preview release/viewer/babylon.glTF2Interface.d.ts
+dist/preview release/viewer/loaders.d.ts
```
Comment (deltakosh) [inline .gitignore]: why not babylon.* ?

Comment (RaananW) [reply .gitignore]: i thought we remove the babylon. . [modulename].* , this is what we discussed

**Comment by RaananW** [comment]
Working on fixing the tests, i just noticed one file didnt rename correctly

File: .gitignore (hunk 1 above)
Comment (deltakosh) [reply .gitignore]: Well misunderstanding then :)  it must be babylon.modulename.* (I think the reason of the misunderstanding is that all modules should be babylon.* for me but the viewer is not following this rule)

**Pull request closed without merging**